src/ngAria/aria.js

```diff
@@ -377,6 +377,25 @@ ngAriaModule.directive('ngShow', ['$aria', function($aria) {
             });
           }
         }
+
+        if (elem[0].nodeName === 'A') {
+          if ($aria.config('bindRoleForClick') && !attr.href && !attr.xlinkHref && !attr.role) {
+              elem.attr('role', 'link');
+          }
+          if ($aria.config('bindKeypress')) {
+            elem.on('keypress', function(event) {
+              var keyCode = event.which || event.keyCode;
+              var hasHref = (attr.href || attr.xlinkHref);
```

Comment:
This is not exactly what we are doing in the built-in `htmlAnchorDirective` (in core) and there's a very slight chance this will not do the expected thing. This bug is so unlikely to _bite_ anyone though, that I'm fine leaving it as is (for the sake of simplicity).